sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -43,6 +44,8 @@ def __init__(self, control_address, worker_count):
     self._worker_count = worker_count
     self._worker_index = 0
     self._control_channel = grpc.insecure_channel(control_address)
+    self._control_channel = grpc.intercept_channel(self._control_channel,
```

Comment:
Should we simplify this as:

`self._control_channel = grpc.intercept_channel(grpc.insecure_channel(control_address), WorkerIdInterceptor())`